src/luminator.ts

```diff
@@ -57,6 +57,17 @@ class Luminator {
     return Math.trunc(crypto.randomBytes(8).readUInt32LE(1) / 0xffffffff * 1000000);
   }
 
+  /**
+   * return status if it is a valid AxiosError
+   * @param error
+   * @return status || -1
+   */
+  private static getStatusFromAxiosError (error: axios.AxiosError): number {
+    const hasStatus: boolean = error.hasOwnProperty('response') && error.response.hasOwnProperty('status');
+
+    return hasStatus ? error.response.status : -1;
```

Comment:
It isn't a good to return -1. Are you sure that it's useful?

Reply:
i was hesitating between 0 and -1, i've chooed the -1 to tell that there is no status (false)